### DISCUSSION TITLE
What is the paper on which U-Net is based on?

### DISCUSSION THREAD

**Discussion by al3ms** [Project-MONAI/MONAI#6534] [Q&A] [answered]
I noticed that there are details provided for each network used in MONAI, specifying the paper on which they are based. However, I couldn't find any specific details about the U-Net model. If I implement U-Net in 3D by setting spatial_dims = 3, could you please provide information about the paper that this implementation is based on? 

Thanks

**Comment by wyli** [comment, marked as answer]
yes please consider citing this paper if you are using the network 
https://github.com/Project-MONAI/MONAI/blob/243fbe63f407f56b9f3315e15b4cf357eebb1351/monai/networks/nets/unet.py#LL35C15-L35C77
